lib/fast-path.js

```diff
@@ -407,6 +407,13 @@ FPp.needsParens = function(assumeExpressionContext) {
       return true;
     }
 
+  case "CallExpression":
+    if (parent.type === "ExportDefaultDeclaration" &&
+        name === "declaration" &&
+        node.callee.type === "FunctionExpression") {
```

Comment:
The test failures seem to be suggesting that `node.callee` could be undefined here, though I'm honestly not sure how that could be the case since `node.type === "CallExpression"`…

Reply:
For some reason `export default {}` in `es6test.js` hits this `CallExpression` case yet clearly contains no calls.  That seems like a pre-existing orthogonal issue.

Using more robust node checking fixes this test case.